x-pack/plugins/lists/public/exceptions/hooks/persist_exception_list.tsx

```diff
@@ -43,7 +46,11 @@ export const usePersistExceptionList = ({
       if (exceptionList != null) {
         try {
           setIsLoading(true);
-          await persistExceptionList({ http, list: exceptionList, signal: abortCtrl.signal });
+          if (isUpdateExceptionList(exceptionList)) {
```

Comment:
NOTE: Same as with `persist_exception_list_item`, separated these out into their own to validate the request/response schemas of each.